src/MonoMod.UnitTest/Temp.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace MonoMod.UnitTest
+{
+    public class Temp : TestBase
+    {
+        public Temp(ITestOutputHelper helper) : base(helper) { }
+
+        [Fact]
+        public void MyThing()
+        {
+            Assert.Equal("abc", "abc".AsSpan().ToArray());
+        }
+    }
```

Comment:
Can we have an actual name for this test?

Reply:
I just deleted it instead, was just using it for debugging, but i can add some tests for general span correctness if you'd like

Reply:
Probably not a bad idea. At some point, we should also add some mini test framework that we can run on .NET 3.5...